src/main/java/com/nasrpi/filesharing/UploadFileResponse.java

```diff
@@ -0,0 +1,57 @@
+package com.nasrpi.filesharing;
+
+/**
+ * Exception class for FileNotFoundException
```

Comment:
is this an exception class ?

Reply:
yes, it is required to handle exceptions occurred while uploading and downloading files.

Reply:
should I resolve this?

Reply:
corrected